ipyleaflet/leaflet.py

```diff
@@ -384,6 +388,27 @@ class Heatmap(RasterLayer):
     gradient = Dict({0.4: 'blue', 0.6: 'cyan', 0.7: 'lime', 0.8: 'yellow', 1.0: 'red'}).tag(sync=True, o=True)
 
 
+class VectorTileLayer(Layer):
+    _view_name = Unicode('LeafletVectorTileLayerView').tag(sync=True)
+    _model_name = Unicode('LeafletVectorTileLayerModel').tag(sync=True)
+
+    url = Unicode().tag(sync=True)
```

Comment:
You are also missing the `vector_tile_layer_styles` option from the Protobuf class.

Reply:
Added thank you